recipe/meta.yaml

```diff
@@ -32,7 +31,7 @@ requirements:
     - pybind11
```

Comment:
Should we ping pybind11? `onnx` 1.13.1 uses pybind11 2.9.2 while our latest is 2.10.4 (as of yesterday).

Reply:
`pybind11 2.9.2` is used only if it's **not** found, see https://github.com/onnx/onnx/blob/ad834eb73ee0cd9b6fa9ea892caeed5fa17d7dc0/CMakeLists.txt#L459-L461. And we do not have **python 3.11 s**upport for `2.9.2`. I'll pin `pybind11` to **2.10.4**